docs/user_guide/hyperopt.md

```diff
@@ -27,6 +30,50 @@ hyperopt:
   metric: loss
 ```
 
+## Default Feature - Level Parameters
+
+In addition to defining hyperopt parameters for individual input or output features (like the `title` feature
+in the example above), feature level parameters can be specified for entire feature types. These parameters will
+follow the same convention as the `defaults` section of the Ludwig config. This is particularly helpful in cases
+where a dataset has a large number of features and you don't want to define parameters for each feature individually.
+
+Feature level parameters are defined using the following keywords in order separated by the `.` delimiter:
```

Comment:
Can you use another phrase for `feature level parameters`? Maybe something like `default feature params`, or something else. As a reader, feature level parameters basically sounds similar to the non-default parameters that users specify otherwise.

Reply:
Also here, it would be nice to have an example of a feature level parameter, like `defaults.text.preprocessing.most_common`, and use the exposition in the list below to refer to what each keyword represents.